StortSpelProjekt/Engine/src/Renderer/TextManager.cpp

```diff
@@ -0,0 +1,274 @@
+#include "stdafx.h"
+#include <vector>
+#include "TextManager.h"
+
+#include "../Renderer/Texture/Texture.h"
+#include "../Renderer/DescriptorHeap.h"
+
+#include "../Misc/Window.h"
+#include "../Misc/AssetLoader.h"
+#include "../Misc/GUI2DElements/Text.h"
+#include "../Misc/GUI2DElements/Font.h"
+
+TextManager::TextManager()
+{
+}
+
+TextManager::~TextManager()
+{
+	for (auto textMap : m_TextMap)
+	{
+		delete textMap.second;
+	}
+	m_TextMap.clear();
```

Comment:
Same as with QuadManager. you delete pointers which may or may not been initialized.